target/linux/ath79/dts/qca9558_tplink_tl-wpa8730-v1.dts

```diff
@@ -0,0 +1,164 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "qca955x.dtsi"
+
+/ {
+	compatible = "tplink,tl-wpa8730-v1", "qca,qca9558";
+	model = "TP-Link WPA8730 v1";
+
+	chosen {
+		bootargs = "console=ttyS0,115200n8";
+	};
```

Comment:
This is in the dtsi, it can be dropped here.

Reply:
dropped